10.2/root/etc/my.cnf.d/utf8.cnf

```diff
@@ -0,0 +1,4 @@
+[mysqld]
```

Comment:
I think, for now and for consistency, lets leave it in, even for 10.2 (since it was only the default in 10.2.2 not earlier version).  We can readdress as 10.3 and beyond roll out.